e2e/tests/regression/test_engine_image.robot

```diff
@@ -0,0 +1,32 @@
+*** Settings ***
+Documentation    Engine Image Test Cases
+
+Test Tags    regression    engine_image
```

Comment:
qq: Are we maintaining `core` test tag in robot also?

Reply:
Yes, the `pytest` test cases tagged with `coretest` will retain the `coretest` tag after being migrated to `robot`.